README.md

```diff
@@ -15,14 +15,14 @@ You'll need a Temasys Account and an App key to use this. [Register here to get
 | -------------- | ---------- | ----------- | --------- | ---------- | ---------- | ---- | ----- |
 | **Platforms:** | Win/Mac/Ubuntu/Android | Win/Mac/Ubuntu/Android | Win/Mac/Ubuntu/Android | Win/Mac | Win | Mac | Win | iOS<br>(`9.x` and below) |
```

Comment:
Ok. I installed it and tested it last month and it didn't work on 10.x, and tested now 10.3.1 and it crashed the moment I opened it. I tested on the office ipad that uses 9.x and it works with getaroom.io.